xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1781,6 +1781,8 @@ void CVideoPlayer::ProcessRadioRDSData(CDemuxStream* pStream, DemuxPacket* pPack
 
 bool CVideoPlayer::GetCachingTimes(double& level, double& delay, double& offset)
 {
+  static bool cache_low_level = true;
```

Comment:
@FernetMenta : I'll have a look at it. I assume, since it the class is quite empty at the moment, in the future more and more stuff will be moved in ProcessInfo ?